lib/commands/cache.js

```diff
@@ -62,20 +64,23 @@ const searchCachePackage = async (path, parsed, cacheKeys) => {
 }
 
 class Cache extends BaseCommand {
-  static description = 'Manipulates packages cache'
+  static description = 'Manipulates packages and npx cache'
   static name = 'cache'
   static params = ['cache']
   static usage = [
     'add <package-spec>',
     'clean [<key>]',
     'ls [<name>@<version>]',
     'verify',
+    'npx ls',
+    'npx rm [<key>]',
+    'npx info [<key>]',
```

Comment:
I typed this up first and never revisited it.  `rm` optionally takes one or more keys, `info` requires one or more keys.